lib/crawly/worker.ex

```diff
@@ -76,16 +76,26 @@ defmodule Crawly.Worker do
              spider_name: atom(),
              response: HTTPoison.Response.t(),
              result: {:ok, {response, spider_name}} | {:error, term()}
-  defp get_response({request, spider_name}) do
+  def get_response({request, spider_name}) do
```

Comment:
as this is an function being exposed for internal use only,  can you add a `@doc false` so that it will not be shown in docs.

Reply:
Done